packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -81,7 +81,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             rowStyleProvider: (row: TimelineChart.TimeGraphRowModel) => ({
                 backgroundColor: 0x979797,// 0xaaaaff,
                 backgroundOpacity: row.selected ? 0.1 : 0,
-                lineColor: 0xdddddd, // hasStates ? 0xdddddd : 0xaa4444, // row.data && row.data.hasStates
+                lineColor: this.props.backgroundTheme === 'light' ? 0xD3D3D3 : 0x3F4146 , // hasStates ? 0xdddddd : 0xaa4444, // row.data && row.data.hasStates
```

Comment:
This is executed once when the row is created. When changing the theme this is not called again until a new row is created. This can happen when the few is closed and reopend or when zooming to different time range where the rows are rebuild with new data from the back-end.  I would suggest to create a new issue tracker to fix this and we work on merging this PR.